tools/buildsys/src/builder.rs

```diff
@@ -511,6 +610,15 @@ fn secrets_args() -> Result<Vec<String>> {
         );
     }
 
+    let ca_bundle_var = "BUILDSYS_CACERTS_BUNDLE";
+    let ca_bundle_path = PathBuf::from(
+        env::var(ca_bundle_var).context(error::EnvironmentSnafu { var: ca_bundle_var })?,
+    );
+
+    if ca_bundle_path.exists() {
+        args.build_secret("file", "ca-bundle.crt", &ca_bundle_path.to_string_lossy());
+    }
```

Comment:
I don't agree with these checks - if the environment variable exists and points to a non-existent path, that should trigger a build failure. Otherwise it's too easy to silently omit a vital piece of configuration and build the wrong thing.

What you want instead is the inverse - if the environment variable is unset or empty, skip the secret; otherwise, assert that the file exists and then pass it as a secret.